forwarder/update_handlers/pva_update_handler.py

```diff
@@ -110,7 +110,11 @@ def _try_to_determine_type(self, response):
                 if response.type()["value"].getID() == "enum_t":
                     is_enum = True
             except AttributeError:
-                self._output_type = numpy_type_from_p4p_type[response.type()["value"]]
+                self._output_type = numpy_type_from_p4p_type[
```

Comment:
Inline comment makes black do ugly things to the formatting, so I think it would be better above the code. Also, the comment could then be made clearer:

"Attribute error raised because getID doesn't exist for array types.
Array output types are prefixed by "a", we don't need this."